challenges/vowel/main.py

```diff
@@ -0,0 +1,16 @@
+#Faça um Programa que verifique se uma letra digitada é vogal ou consoante.
+
+
+
+
+alpha = input("type a letter here")
+
+if (alpha == 'a'or alpha == 'e'or alpha == 'i'or alpha =='o' or alpha == 'u'):
+    print("the letter you typed is vowel")
+
+elif (alpha == "w" or alpha == "y"):
+    print("the letter you typed is semi vowel")
+
+else:
+    print("the letter you typed is consonant")
+    
```

Comment:
This code works, but can be significantly improved to let is "more Pythonic".

The semi vowel here is not meant to be implemented. Vowels an consonants are enough :)

```suggestion
VOWELS = ["a", "e", "i", "o", "u"]

if alpha in VOWELS:
    print("the letter you typed is vowel")

else:
    print("the letter you typed is consonant")
 
```